src/index.js

```diff
@@ -1,11 +1,39 @@
-const screenChoise = document.getElementById('screenChoice');
+const screenChoice = document.getElementById('screenChoice');
+const hotdogRadioBtns = document.getElementsByName('hotdog');
+const popcornRadioBtns = document.getElementsByName('popcorn');
+const drinkRadioBtns = document.getElementsByName('drink');
+const submitBtn = document.getElementById('submitBtn');
 
 let screenType = '';
+let priceForHotdog = 0, priceForPopcorn = 0, priceForDrink = 0;
 
-screenChoise.addEventListener('change', () => {
-     checkScreenType(screenChoise.value);
+screenChoice.addEventListener('change', () => {
+     checkScreenType(screenChoice.value);
 });
 
+for (let i = 0; i < hotdogRadioBtns.length; i++) {
```

Comment:
Would this work? `for(let btn of hotdogRadioBtns) { ... }`

Reply:
Yeah, I think it will work, do I need to change it or it's ok like this?

Reply:
It's ok the way it is, just was thinking how it can be made simpler :)